src/mon/OSDMonitor.cc

```diff
@@ -3510,15 +3511,33 @@ bool OSDMonitor::preprocess_boot(MonOpRequestRef op)
     const auto &otype = type_iter->second;
     // m->metadata["osd_type"] must be "crimson", classic doesn't send osd_type
     if (otype == "crimson") {
-      if (!osdmap.get_allow_crimson()) {
-	mon.clog->info()
-	  << "Disallowing boot of crimson-osd without allow_crimson "
-	  << "OSDMap flag.  Run ceph osd set-allow-crimson to set "
-	  << "allow_crimson flag.  Note that crimson-osd is "
-	  << "considered unstable and may result in crashes or "
-	  << "data loss.  Its usage should be restricted to "
-	  << "testing and development.";
-	goto ignore;
+      cmdmap_t cmdmap;
```

Comment:
A little bit doubtful about how I'm initialising the cmdmap here and if it's sufficient to get the crimson attribute.